### ISSUE TITLE
Test mode notice being displayed twice

### ISSUE THREAD

**Issue by ricardo** [Automattic/woocommerce-payments#1604]
The test mode notice: `Test mode active: All transactions are simulated. Customers can't make real purchases through WooCommerce Payments.` is being displayed twice in the Payments > Settings page.

![image](https://user-images.githubusercontent.com/5509901/115061688-de45f180-9ebf-11eb-8c7b-92f74ff2e62a.png)

**Comment by jrodger** [comment]
It looks like the issue was introduced when we added the SEPA gateway [here](https://github.com/Automattic/woocommerce-payments/pull/1537/files#diff-57ee486c4a3ba4bae478eb31255c9a968b5b4eccd04b27f512e4003440d6f821R420-R421) ([PR](https://github.com/Automattic/woocommerce-payments/pull/1537)).

We now create 2 class instances that extend `WC_Payment_Gateway_WCPay`. That causes the notice code to run twice because we set all that up in the constructor.

Usually I'd fix this by making sure the constructor doesn't have these sort of side-effects, but then that raises the question of where this notice code should live.

**Comment by kalessil** [comment]
We should move this to `\WC_Payments::init`, gateways are instantiated there so test mode checks will be available there.

**Comment by jrodger** [comment]
I've created #1613 to resolve this particular issue and created a new issue #1614 to handle the more general case. We don't need to worry about #1614 immediately because once the fix for this issue is merged then everything will be behind a feature flag.
